expectise/expect.py

```diff
@@ -233,10 +238,24 @@ def set_up(cls, klass: Type, method_name: str, surrogate: Callable) -> None:
             "decorators": [],
             "with_args": [],
             "return": [],
+            "mock_ref": mock_ref,
```

Comment:
suggestion for simplicity/brevity: `"mock"` instead of `"mock_ref"` ?

Reply:
and we may be able (in a future step) to get the `surrogate` from the `mock` object?

Reply:
Good point about the variable naming, changed.
Also I don't think I understand what you mean by the second point - surrogate is already a property of the Mock object, so you're already able to obtain it with mock.surrogate.

Or do you mean that we should't explicitly store the surrogate, but rather instead of doing `self.set_method(Expect.method_h[key]["method"])` (in the `to_receive function`, line 130 of `expect.py`), we instead can do:

`self.set_method(Expect.method_h[key]["mock"].surrogate)`

Reply:
Implemented the above, let me know if that's better otherwise I can revert it